### ISSUE TITLE
Add an equation to the Linear Regression output

### ISSUE THREAD

**Issue by cmontgo** [silkyproject/silkyR#399]
Wondering if it would be possible to add an equation output (i.e. a toggle and/or like in the output with Dependent=Int+B covariate...) with the Linear Regression analysis. For simple linear regression it helps students learning, and for larger models I think it helps see what's going on a little. Possibly a simple addition?

**Comment by raviselker** [comment]
Would definitly be a cool addition! I'll have a think about how and when we could implement his. It's always a priorities game :)

**Comment by cmontgo** [comment]
Of course. I wasn’t thinking anything fancy, just a line with Y(name if easy) = int+ Σ \beta variables. Just mostly for students learning regression to easily relate it to the table etc
